src/Voron/Platform/Posix/PosixHelper.cs

```diff
@@ -129,5 +130,11 @@ public static void EnsurePathExists(string file)
             }
             dirsToCreate.ForEach(x => Directory.CreateDirectory(x));
         }
+
+        public static void EnsureRWPermissionsForOwnerAndGroup(string filePath)
+        {
+            var fileInfo = new UnixFileInfo(filePath);
+            fileInfo.FileAccessPermissions = fileInfo.FileAccessPermissions | FileAccessPermissions.UserRead | FileAccessPermissions.UserWrite | FileAccessPermissions.GroupRead | FileAccessPermissions.GroupWrite;
```

Comment:
@ppekrol is that how we do it in RavenDB? Mono.Unix namespace?

Reply:
Try https://learn.microsoft.com/en-us/dotnet/api/system.io.file.setunixfilemode?view=net-8.0

Reply:
It was introduced in .Net 7. IIRC we have still do a custom release of v5.4 on .Net 6.

Reply:
Use .NET impl, I will deal with .NET 6 if needed